### PR TITLE
Updates to governance, bylaws, principles and boards.

### PR DESCRIPTION
**Pull request by bdc34**
Changes edited by Stephanie Orphan.

### CODE REVIEW

File: source/about/governance.md
Comment (sdorphan) [inline source/about/governance.md:1]: in line 3, "operating principle" should be "operating principles"

**Review by jimentwood — APPROVED** [review]
Looks good

File: source/about/governance.md
Comment (bdc34) [reply source/about/governance.md:1]: Fixed.

File: source/about/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ arXiv offers researchers a broad range of services: article submission, compilat
...
 
-Operations are maintained by the [arXiv Leadership Team](people/leadership_team.md) and arXiv staff at Cornell. Governance of arXiv is led by the Leadership Team with guidance from the arXiv Scientific Advisory Board and the arXiv Member Advisory Board. arXiv is a community-supported resource funded by [Cornell University](https://www.cornell.edu/), the [Simons Foundation](https://www.simonsfoundation.org), [member institutions](ourmembers.md), and [donors](give.md).
+Operations are maintained by the [arXiv Leadership Team](people/leadership_team.md) and arXiv staff at Cornell. Governance of arXiv is led by the Leadership Team with guidance from the arXiv [Editorial Advisory Council](people/editorial_advisory_council.md), [Institutions Advisory Council](people/institutions_advisory_council.md), and [Science Advisory Council](people/science_advisory_council.md). arXiv is a community-supported resource funded by [Cornell University](https://www.cornell.edu/), the [Simons Foundation](https://www.simonsfoundation.org), [member institutions](ourmembers.md), and [donors](give.md).
```
[the remark this answers is not in the document]
Comment (bdc34) [reply source/about/index.md]: Fixed.

File: source/about/people/member_ad_board.html
Comment (SBBCornell) [inline source/about/people/member_ad_board.html:1]: similar to mab_bylaws.html, just wondering about the html extension rather than .md.

**Review by SBBCornell — COMMENTED** [review]
Seems some file extensions were changed to html. Other than that, the content changes look fine and I dont think conflict with other changes made to the main docs.

File: source/about/mab_bylaws.html
[the remark this answers is not in the document]
Comment (bdc34) [reply source/about/mab_bylaws.html:1]: mab_bylaws.html is now a redirect and this is the current technique used for redirects in arxiv-docs.

**Review by SBBCornell — APPROVED** [review]
looks good

**Review by sdorphan — APPROVED** [review]
Looks good.

**Pull request merged**